README.md

```diff
@@ -1,6 +1,7 @@
 # QA Test Automation (QATA) framework
 for UC Berkeley SIS Campus Solutions
 
+[![Build Status](https://travis-ci.org/mDibyo/sis-qa-test-auto.svg?branch=SISRP-10794)](https://travis-ci.org/mDibyo/sis-qa-test-auto)
```

Comment:
See my comment above. It did not make sense to put in the master branch (or the ucberkeley repo) for the review because there have been no Travis builds on that branch. I will change it now.